wormhole-connect/src/views/v2/Bridge/AmountInput/index.tsx

```diff
@@ -207,7 +207,6 @@ const AmountInput = (props: Props) => {
                 e.currentTarget.blur();
               },
               step: '0.1',
-              pattern: '[0-9]+([.|,][0-9]{1,2})?',
```

Comment:
We don't need this anymore as we are checking the input type/format in the inner Input control.